docs/_data/components/schemas/limit-provenance.yaml

```diff
@@ -3,36 +3,61 @@ description: Given a limit value, a set of data defining the provenance of that
 properties:
   proposals-considered:
     description: |
-      Set of IDs of proposals that affect this limit.  
+
+      The forecast proposals provided by the Ratings Providers during the
+      Forecast Window for this limits snapshot.
+
     type: array
-    maxItems: 10
+    maxItems: &max-proposals 10
     items:
-      $ref: ./generic-identifier.yaml
+      allOf:
+        - $ref: 'forecast-limit-period.yaml#/proposal-considered'
+        - type: object
+          properties:
+            resource-id:
+              $ref: ./resource-id.yaml
+
   temporary-aar-exceptions:
     description: |
-      Set of IDs temporary AAR exceptions that affect this limit.  
+
+      The temporary AAR exceptions for the facility that were active when this
+      snapshot was generated.
+
+    type: array
+    maxItems: *max-proposals
+    items:
+      $ref: ./resource-id.yaml
+
+  overrides:
     type: array
```

Comment:
This should be very uncommon, but I thought it would be appropriate to allow more than one override to be associated with a limit, since overrides are available at the SEG and DYNELE level are possible. Ultimately, all of this provenance data is informational/archival and not used to compute anything, so listing things feels safe and appropriate.

Reply:
I think this is OK.

Reply:
We agreed an array of overrides is appropriate, and in that discussion confirmed we were aligned on the purposes of the provenance/override/aar exception info in the detailed snapshot. It is specifically not meant to allow for a clearinghouse re-run, but it does support troubleshooting and post facto analysis of how limits were determined. The salient distinction is that you don't need a pointer that says "this proposal's continuous rating proposal was used" or "this proposal's emergency duration 30 minute rating was used".